src/nidmm/metadata/functions_addon.py

```diff
@@ -101,16 +102,16 @@
                                                    2: { 'default_value': False, },
                                                    3: { 'default_value': '""', }, }, },
     'ConfigureMultiPoint':       { 'parameters': { 3: { 'default_value': 'SampleTrigger.IMMEDIATE', },
-                                                   4: { 'default_value': -1, }, }, },
+                                                   4: { 'default_value': 'datetime.timedelta(seconds=-1)', }, }, },
```

Comment:
Does this work as expected, meaning, the DLL receives a -1? I suggest adding a unit test to nifake to make sure.

Reply:
Yes, this works correctly.

Added explicit -1 tests.